pkg/security/certs_tenant_test.go

```diff
@@ -81,18 +81,19 @@ func TestTenantCertificates(t *testing.T) {
 func testTenantCertificatesInner(t *testing.T, embedded bool) {
 	defer leaktest.AfterTest(t)()
 
-	var tenant, certsDir string
+	var certsDir string
+	var tenant uint64
 	if !embedded {
 		// Don't mock assets in this test, we're creating our own one-off certs.
 		security.ResetAssetLoader()
 		defer ResetTest()
-		tenant = fmt.Sprint(rand.Int63())
+		tenant = rand.Uint64()
```

Comment:
I think we'll restrict tenant IDs to `MaxInt64`, so consider sticking with `rand.Int63n`. Or - just hard-code something, not sure this use of rand is very useful.

Reply:
done